components/demo/demo-snippet.js

```diff
@@ -35,12 +34,8 @@ class DemoSnippet extends LitElement {
 			}
 			.d2l-demo-snippet-demo {
 				flex: 1 1 auto;
-				overflow: hidden;
 				position: relative;
 			}
-			:host([allow-overflow]) .d2l-demo-snippet-demo {
-				overflow: visible;
-			}
```

Comment:
Going back on these changes as they clipped tooltips and stuff -- just made a change to the scroll-wrapper demo to do this a different way.